airbyte-cdk/python/airbyte_cdk/destinations/vector_db_based/embedder.py

```diff
@@ -79,8 +90,8 @@ def check(self) -> Optional[str]:
             return format_exception(e)
         return None
 
-    def embed_texts(self, texts: List[str]) -> List[List[float]]:
```

Comment:
Should this be renamed to `embed_chunks`?

Reply:
yes, updated